docs/documentation_routes.js

```diff
@@ -77,6 +77,10 @@ router.get('/make-first-prototype/add-questions', function (req, res) {
   res.redirect('/docs/make-first-prototype/use-components')
 })
 
+router.get('/docs/views/templates/check-your-answers', function (req, res) {
```

Comment:
its confusing, but the website path does not include `views`, so that needs to be removed from both paths

Reply:
@joelanman Thanks for flagging! Will change both and then re-request a review from you if that's ok.
```suggestion
router.get('/docs/templates/check-your-answers', function (req, res) {
```